projects/batfish-common-protocol/src/org/batfish/datamodel/Interface.java

```diff
@@ -173,17 +187,22 @@ else if (name.startsWith("TenGigabitEthernet")) {
          return InterfaceType.PHYSICAL;
       }
       else if (name.startsWith("Tunnel")) {
-         return InterfaceType.VPN;
```

Comment:
According to http://www.cisco.com/en/US/technologies/tk583/tk372/technologies_white_paper0900aecd8029d629_ps6635_Products_White_Paper.html tunnel interfaces can be used for IPSec VPNs - will rollback this change.